sdk/communication/communication-phone-numbers/test.env

```diff
@@ -0,0 +1,10 @@
+# Used in test to tetrieve these values from a Communication Services resource
+# in the Azure Portal.
+AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING="endpoint=<endpoint>;accessKey=<accessKey>"
```

Comment:
How does `test.env` get picked up? Is this a new change in the live test framework?

Reply:
This is new guidance from migration guide. The `sample.env` should have sample env variables used in the samples. Since our tests use different env variables from our samples I had to move them to a `test.env`.

![image](https://user-images.githubusercontent.com/13181430/113617323-0e150f80-960b-11eb-80b4-a363a768ccaa.png)

Reply:
These files are merely templates. At runtime the configuration is read from `.env`, and sample.env/test.env are merely meant to serve as points of documentation for developers of the required environment configuration to run the tests/samples.